tests/test_indexer.py

```diff
@@ -52,6 +54,16 @@
 #
 
 
+class ElasticsearchTestClient:
+    _es_client = None
+
+    @staticmethod
+    def get(logger):
+        if ElasticsearchTestClient._es_client is None:
+            ElasticsearchTestClient._es_client = connect_elasticsearch(os.getenv("DSS_ES_ENDPOINT"), logger)
+        return ElasticsearchTestClient._es_client
```

Comment:
Non-blocking comment: this construct seems overwrought and confusing to me. Logger is a global and can just be ingested out-of-band. At first glance, `client.get(logger)` appears to do something with logging, not a database connection manager.

In a test case, I would just use a setUpClass method to initialize a connection manager as a class variable.

Let's revisit this later.